README.md

```diff
@@ -169,6 +169,7 @@ If you use multiline logging with raw, it's recommended to json encode the Data
 * `DEBUG` - emit debug logs
 * `EXCLUDE_LABEL` - exclude logs with a given label
 * `INACTIVITY_TIMEOUT` - detect hang in Docker API (default 0)
+* `HTTP_BIND_ADDRESS` - configure which address to listen on (default 0.0.0.0)
```

Comment:
please change to:
`HTTP_BIND_ADDRESS` - configure which address/interface to listen on (default 0.0.0.0)